### ISSUE TITLE
clean kruft

### ISSUE THREAD

**Issue by rsignell-usgs** [ioos/notebooks_demos#169]
I just cloned the repo and noticed it's over 50mb.  Probably time to clean the old useless notebook versions out.   @ocefpaf, I thought we had figured out a nice way to do this, but I couldn't find it...

**Comment by ocefpaf** [comment]
I understand that but at the same time it was nice to have all the previous run on the history  when you wrote the paper :wink: 

Cleaning per se is not complicated. What comes next is: (1) we need to contact GitHub to run the "garbage collector" to actually remove the files we removed from the history. Git does that automatically at some point but if we do not contact them then we re-introduce the files in the history because of our forks. (2) as soon as the cleanup happens we need to re-fork the project.

Coordinating (1) and (2) is not that easy. I cleaned `pysgrid` 3 times now and someone always re-introduces the files from their forks because they fail to do (2) :unamused: 

TL;DR

I am +0 to this change. Will do if you really think it is necessary but I do not consider it an issue.

**Comment by rsignell-usgs** [comment]
@ocefpaf, you make good points here.  Okay, closing.
